### PR TITLE
Formatting updates

### PR DESCRIPTION
**Pull request by localastronaut**
Made some formatting updates. Since they were mostly pretty small updates I did them on the same PR just to avoid conflicts.

### CODE REVIEW

File: static/css/artist/artist_preview.css [hunk 1]
```diff
@@ -18,3 +18,22 @@
...
+    margin: 10px;
+    padding: 15px;
+    width: 120px;
+    background-color: #4A96AD;
+}
+
+.view-artists-box {
+    width: 200px;
+    font-size: 1.5em;
+    font-family: dosis;
```
Comment (RevolutionTech) [inline static/css/artist/artist_preview.css]: I think `dosis` should be title case: `Dosis`.

File: static/css/home.css [hunk 2]
```diff
@@ -109,3 +109,32 @@ p.welcome-text {
 .icon-padding {
     padding: 20px;
 }
+
+.view-artists-box {
+    width: 200px;
+    font-size: 1.5em;
+    font-family: dosis;
```
Comment (RevolutionTech) [inline static/css/home.css]: Capital `D` here as well.

File: static/css/perdiem.css [hunk 3]
```diff
@@ -4,7 +4,7 @@ h1, h2, h3, h4, h5, h6, p, li, input [type=submit] {
 }
 
 h4 [type=submit] {
-    font-family: 'Roboto', sans-serif;
+    font-family: 'roboto', sans-serif;
```
Comment (RevolutionTech) [inline static/css/perdiem.css]: Why the change to lowercase?

File: templates/artist/artist_list.html [hunk 4]
```diff
@@ -13,9 +13,9 @@
     </div>
     <div class="filter-sort-options center">
         <ul class="dropdown menu" data-dropdown-menu data-options="disableHover:true; clickOpen:true;">
-            <li><h5>Show me</h5></li>
```
Comment (RevolutionTech) [inline templates/artist/artist_list.html]: I think our CSS has an `h5` class that's not being used anymore and we can get rid of.

File: templates/artist/includes/artist_updates.html [hunk 5]
```diff
@@ -1,12 +1,16 @@
 <ul class="menu vertical nested is-active updates">
     {% for update in updates %}
         <li id="update-{{ update.id }}">
-            <h3>
-                {% if show_artist_name %}{{ update.artist.name }}: {% endif %}{{ update.title }}
+            <h2>
```
Comment (RevolutionTech) [inline templates/artist/includes/artist_updates.html]: This should be formatted as:
```html
{% if show_artist_name %}
    <h2>{{ update.artist.name }}</h2><br />
{% endif %}
<h4>
    {{ update.title }}
    {% if has_permission...
    ....
</h4>
```

File: templates/base.html [hunk 6]
```diff
@@ -53,7 +53,7 @@
                         <div class="hide-for-small-only top-bar nav">
                             <div class="top-bar-left">
                                 <ul class="dropdown menu nav" data-dropdown-menu>
-                                    <li><a class="active" href="{% url 'home' %}">HOME</a></li>
```
Comment (RevolutionTech) [inline templates/base.html]: I think `active` is a class from Foundation that is applied to whichever link is the one that's currently selected. Are you want to remove this?

File: templates/home.html [hunk 7]
```diff
@@ -21,68 +21,75 @@ <h1 class="home-title">PERDIEM</h1>
...
+                <a class="button text-center box-margin view-artists-box" href={% url 'artists' %}>START INVESTING</a>
             </div>
         {% endif %}
         <div class="bottom-how">
             <h5><a class="white-color" href={% url 'home' %}#how>HOW IT WORKS<br/>
             <i class="fa fa-chevron-down fa-1x" aria-hidden="true"></i></a></h5>
         </div>
     </div>
-    <!-- How it works for fans -->
+        <!-- How it works for fans -->
```
Comment (RevolutionTech) [inline templates/home.html]: Remove spaces before comment here

File: templates/home.html [hunk 8]
```diff
@@ -21,68 +21,75 @@ <h1 class="home-title">PERDIEM</h1>
         <h2 class="home-description">THE NEXT GENERATION OF MUSIC</h2>
         {% if user.is_authenticated %}
             <div class="home-boxes">
```
Comment (RevolutionTech) [inline templates/home.html]: Since both authenticated and unauthenticated users see the same thing here, there's no need for the `if` statement. We can just replace this with:
```
<div class="home-boxes">
    <a class="button text-center box-margin view-artists-box" href="{% url 'artists' %}">START INVESTING</a>
</div>
```

Also watch out, it looks like the quotes around the URL in the `a` tag are missing in your change.

File: static/css/perdiem.css (hunk 3 above)
Comment (localastronaut) [inline static/css/perdiem.css]: I noticed that on safari it wouldn't load (locally) the font unless it was in a lowercase "robot". The other strange thing I noticed was that it worked on the beta even though it didn't work on my local machine. Not really sure why that was happening.

Comment (RevolutionTech) [inline static/css/perdiem.css]: Oh okay, that's super weird. Have you tried loading the fonts from base.html as lowercase?

File: templates/home.html [hunk 9]
```diff
@@ -21,68 +21,75 @@ <h1 class="home-title">PERDIEM</h1>
...
+        <div class="video-padding">
+            <div class="videowrapper">
+                <iframe width="1280" height="720" src="https://www.youtube.com/embed/M78Ep_qcnpA" frameborder="0" allowfullscreen></iframe>
+            </div>
+        </div>
+        <h4 class="more-extra-padding">WE HELP ARTISTS GET FUNDING BY HAVING FANS INVEST IN THEIR MUSIC</h4>
+        <div class="break">
+            <hr>
+        </div>
+        <h3 class="how-title">HOW IT WORKS FOR ARTISTS</h3>
```
Comment (RevolutionTech) [inline templates/home.html]: I'd be curious to understand why you decided to reorder the artists section before fans. I think having fans first makes sense, since naturally the majority of users on the site will be fans.

File: templates/home.html [hunk 10]
```diff
@@ -21,68 +21,75 @@ <h1 class="home-title">PERDIEM</h1>
...
+                <h3>GIVE</h3>
+                <p class="how-extra">GIVE A PERCENTAGE OF SALES BACK TO PEOPLE WHO INVEST IN YOUR MUSIC PROJECT</p>
             </div>
             <div class="icons">
                 <i class="fa fa-dollar fa-4x icon-padding" aria-hidden="true"></i>
                 <h3>EARN</h3>
-                <p class="how-extra">EARN MONEY BASED ON HOW SUCCESSFUL THE PROJECT IS</p>
+                <p class="how-extra">EARN MONEY BY BUILDING A TEAM OF PEOPLE ALL PUSHING FOR YOUR SUCCESS</p>
             </div>
-        </div>
```
Comment (RevolutionTech) [inline templates/home.html]: Looks like this line was removed accidentally?

File: templates/home.html [hunk 11]
```diff
@@ -21,68 +21,75 @@ <h1 class="home-title">PERDIEM</h1>
...
+
     <!-- How it works for artists -->
```
Comment (RevolutionTech) [inline templates/home.html]: nit: Indent comment in

File: templates/leaderboard/includes/list.html [hunk 12]
```diff
@@ -4,16 +4,39 @@
...
-            </li>
-        {% endfor %}
-    </ul>
+    <div class="center">
+        <div class="icons">
+            <h3>{{ name }}</h3>
+            <ul>
+                {% for investor, profile_context in investors %}
+                    <li>
+                    <a href="{{ investor.public_profile_url }}">
```
Comment (RevolutionTech) [inline templates/leaderboard/includes/list.html]: nit: Indent this section
```html
<li>
    <a href="...">
        <div class="leaderboard-padding button">
            <img ...>
            ...
        </div>
    </a>
</li>
```

File: templates/leaderboard/includes/list.html [hunk 13]
```diff
@@ -4,16 +4,39 @@
...
+                        <br/>
+                        <h5 class="investor-name">#{{ forloop.counter }}: {{ investor.get_display_name }}</h5>${{ profile_context.total_earned|floatformat:2|intcomma }}
+                    </div>
+                    </a>
+                    </li>
+                {% endfor %}
+            </ul>
+        </div>
+
+        <div class="icons">
```
Comment (RevolutionTech) [inline templates/leaderboard/includes/list.html]: Why two of these?

File: templates/artist/artist_list.html [hunk 14]
```diff
@@ -42,8 +42,8 @@
             {% endif %}
             <li>
                 <button class="button hollow location" id="location-dropdown-button" type="button" data-toggle="location-dropdown">
```
Comment (RevolutionTech) [inline templates/artist/artist_list.html]: I think the padding on this button is too small now.

File: templates/artist/artist_list.html [hunk 15]
```diff
@@ -24,11 +24,11 @@
...
                 </ul>
             </li>
-            <li><h5>artists in</h5></li>
+            <li><h6>artists in</h6></li>
             {% if genres %}
                 <li class="is-dropdown-submenu-parent opens-right">
-                    <a href="#"><h5>{{ active_genre }}</h5></a>
-                    <ul class="menu is-dropdown-submenu">
+                    <a href="#"><h6>{{ active_genre }}</h6></a>
+                    <ul class="menu is-dropdown-submenu>
```
Comment (RevolutionTech) [inline templates/artist/artist_list.html]: It looks like the close quote is missing after `is-dropdown-submenu`. I think that's what's preventing the first option from not being clickable.

File: static/css/perdiem.css (hunk 3 above)
Comment (localastronaut) [inline static/css/perdiem.css]: Okay it's working in uppercase now. Not sure why it was doing that before.

File: templates/artist/artist_list.html (hunk 4 above)
Comment (localastronaut) [inline templates/artist/artist_list.html]: Using h5 in a the "extra" pages

File: templates/home.html (hunk 9 above)
Comment (localastronaut) [inline templates/home.html]: I agree. Not sure why that happened.

File: templates/leaderboard/includes/list.html (hunk 13 above)
Comment (localastronaut) [inline templates/leaderboard/includes/list.html]: One of them was going to be "Total Earned" for Artists. I remember you mentioned it wouldn't be too difficult to add this one so went ahead and created a section for it.

Comment (RevolutionTech) [inline templates/leaderboard/includes/list.html]: Oh yeah, that's right. Okay, cool. I'll apply a commit to this branch with that change.

File: templates/home.html [hunk 16]
```diff
@@ -19,26 +19,31 @@
         <div class="heading-center"></div>
         <h1 class="home-title">PERDIEM</h1>
         <h2 class="home-description">THE NEXT GENERATION OF MUSIC</h2>
-        {% if user.is_authenticated %}
             <div class="home-boxes">
```
Comment (RevolutionTech) [inline templates/home.html]: nit: Unindent this section now that the `if` statement has been removed.

File: templates/home.html [hunk 17]
```diff
@@ -19,23 +19,19 @@
...
         <div class="bottom-how">
             <h5><a class="white-color" href={% url 'home' %}#how>HOW IT WORKS<br/>
             <i class="fa fa-chevron-down fa-1x" aria-hidden="true"></i></a></h5>
         </div>
     </div>
-        <!-- How it works for fans -->
+
     <div id="how-page" class="page">
         <a id="how" class="smooth"></a>
+
```
Comment (RevolutionTech) [inline templates/home.html]: Why the extra newline here?
